content/slack/index.md

```diff
@@ -61,24 +61,33 @@ For future enhancements to the tool, see the [`f3_bot` issues](https://github.co
 
 {{< button relref="/articles/paxmate-user-guide" >}}Paxmate User Guide{{< /button >}} | {{< button relref="/paxmate-dashboard" >}}Paxmate Dashboard{{< /button >}}
 
+## Profiles
+
+Here are some tips for filling out your profile.
+
+- Most people have a silly profile picture related to their nickname, but you are also welcome to use a selfie.
+- For your **Display Name**, use your F3 name. Some people from other regions might also add their region, like **Prodigal (Carpex)**.
+- In your title, feel free to add what you do in real life, or in the group, like if you're a Site Q.
+- Make sure to add a phone number to the **ICE**, or in case of emergency, field. You can also put this in your title if you want to add more details like a contact name. (Note: You can fill out the **ICE** field on desktop or browser only, unfortunately not on the mobile app yet due to a known Slack limitation.)
+
 ## Moderation
 
-As with everything in F3, you are "Freed to Lead," and can post mostly whatever you like, or even help the [Commz team](/leadership/) moderate.
+As with everything in F3, you are "Freed to Lead," and can post mostly whatever you like, or even help the [Commz team](/leadership/) moderate. We want to make sure that our digital community reflects the values of F3 and the spirit of the group that you get in person. The following guidelines are meant to help you understand the sort of things we look for.
```

Comment:
Preview of this section

<img width="851" alt="image" src="https://github.com/user-attachments/assets/1cc44e3d-52dd-40db-986d-ba1b7954267f" />